src/content/docs/nrql/using-nrql/create-smoother-charts-sliding-windows.mdx

```diff
@@ -79,27 +79,29 @@ However, in this example, 5-minute wide `TIMESERIES` windows are smoothed with t
   TIMESERIES query with SLIDE BY clause
 </figcaption>
 
-## Valid NRQL syntax for `SLIDE BY` [#valid-syntax]
+## `SLIDE BY` syntax [#valid-syntax]
 
-Valid NRQL syntax for the `SLIDE BY` clause will follow the format below.
+Valid NRQL syntax for the `SLIDE BY` clause follows the format below.
```

Comment:
Much better :) see how dropping the "will" makes the sentence feel more direct and concise?